webrtc.html

```diff
@@ -385,7 +385,7 @@
         the offer/answer creation process.</p>
 
         <dl class="idl" title="dictionary RTCOfferAnswerOptions">
-          <dt>boolean voiceActivityDetection = true</dt>
+          <dt>RTCVoiceActivityOption voiceActivityDetection = "enabled"</dt>
 
```

Comment:
Can we do the same "or boolean" trick here as with PR 471 to make it more convenient?